gui/src/components/home/ResetButton.tsx

```diff
@@ -173,6 +192,13 @@ export function ResetButton({
     };
   }, []);
 
+  if (
+    (bodyPartsToReset === 'fingers' && !hasFingerTrackers) ||
+    (bodyPartsToReset === 'feet' && !hasFeetTrackers)
+  ) {
+    return null;
+  }
```

Comment:
This will be weird if we have some kinda step that uses this button and the button just vanishes lol, maybe the parent should control it? or like have it disabled by this?